setup.py

```diff
@@ -0,0 +1,49 @@
+"""Setup rio-stac."""
+
+from setuptools import find_packages, setup
+
+with open("README.md") as f:
+    long_description = f.read()
+
+inst_reqs = [
+    "rasterio",
+    "pystac>=0.5,<0.6",
+]
+
+extra_reqs = {
+    "dev": ["pytest", "pytest-cov", "pre-commit", "requests"],
+    "test": ["pytest", "pytest-cov", "requests"],
+}
+
+
+setup(
+    name="rio-stac",
+    version="0.1.0",
+    description=u"Create STAC Items from a raster data.",
```

Comment:
```suggestion
    description="Create STAC Items from raster data.",
```
The `u` is from Python 2; you don't need it?